gradle/validation/validate-log-calls.gradle

```diff
@@ -118,7 +118,7 @@ class ValidateLogCallsTask extends DefaultTask {
 
     if (hasPlus) {
       cause = "hasPlus: " + hasPlus
-      violation = true
+      violation = level != "error"
```

Comment:
What's this about?  Could use a comment.

Reply:
Yeah, this was a misunderstanding of the log.error() function, and the fact that the last parameter is treated as an exception and not a parameter to the string format. I can undo this